components/json/rs/src/lib.rs

```diff
@@ -0,0 +1,128 @@
+#[cxx::bridge(namespace = json)]
+mod ffi {
+    extern "Rust" {
+      fn convert_uint64_to_unicode_string(path: &str, json: &str) -> String;
+      fn convert_int64_to_unicode_string(path: &str, json: &str) -> String;
+      fn convert_unicode_string_to_uint64(path: &str, json: &str) -> String;
+      fn convert_unicode_string_to_int64(path: &str, json: &str) -> String;
+    }
+}
+
+// Converts uint64 value to string and serializes json with new values.
+// A path is a Unicode string with the reference tokens separated by /. Inside tokens / is replaced by ~1 and ~ is replaced by ~0.
+// Examples: convert_uint64_to_unicode_string("/a/b", json) for { a : { b : 1 }}
+//           convert_uint64_to_unicode_string("/a/0", json) for { a : [ 1 ]}
+pub fn convert_uint64_to_unicode_string(path: &str, json: &str) -> String {
+  let mut unwrapped_value: serde_json::Value = serde_json::from_str(&json).unwrap_or_else(|e| {
```

Comment:
This can just be `::from_str(json)`. The argument is already a reference, so you don't need to double-reference it.

Reply:
done